.mergify.yml

```diff
@@ -0,0 +1,31 @@
+pull_request_rules:
+  - name: ask to resolve conflict
+    conditions:
+      - conflict
+    actions:
+        comment:
+          message: |
+            This pull request is now in conflicts. Could you fix it @{{author}}? 🙏
+            To fixup this pull request, you can check out it locally. See documentation: https://help.github.com/articles/checking-out-pull-requests-locally/
+            ```
+            git fetch upstream
+            git checkout -b {{head}} upstream/{{head}}
+            git merge upstream/{{base}}
+            git push upstream {{head}}
+            ```
+  - name: backport patches to 7.x branch
+    conditions:
+      - base=master
+      - label=v7.13.0
```

Comment:
We will need some automation in the future to bump the version when required

Reply:
Just so I am sure I understand how this works. If we set the `v7.13.0` on the PR, it will be auto backported?

Reply:
Exactly, but only when it gets merged.

To illustrate the above, please see the below example for one of the repositories where this service is configured:

1) https://github.com/elastic/apm-pipeline-library/pull/994 contains the label `backport-to-7.x` (that's what we are using in that particular repo), and was merged.
2) A mergify action was triggered -> https://github.com/elastic/apm-pipeline-library/runs/1920298071
3) https://github.com/elastic/apm-pipeline-library/pull/995 was created afterwards

Reply:
Great!